### PR TITLE
Mariocube.DTools version 1.0.0.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103187)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230422171231

- Is there an installer switch to bypass this and install automatically?

**Comment by upintheairsheep** [comment]
> Automatic Validation ended with: InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230422171231
> 
> * Is there an installer switch to bypass this and install automatically?

Try common flags

**Comment by upintheairsheep** [comment]
and if that fails, Interactive-Only-Installer

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Review by ItzLevvie — DISMISSED** [review]

**Comment by stephengillie** [comment]
image

File: manifests/m/Mariocube/DTools/1.0.0.0/Mariocube.DTools.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.DTools
+PackageVersion: 1.0.0.0
+Installers:
+- InstallerUrl: https://repo.mariocube.com/Wii/PC%20Applications/DTools/DTools%20Setup.exe
+  Architecture: x86
+  InstallerType: exe
+  InstallerSha256: 1BAC57215ADE599D1619BB1385EC74717C9261C4C00A1A24B6E2073B741F7997
```
Comment (stephengillie) [inline manifests/m/Mariocube/DTools/1.0.0.0/Mariocube.DTools.installer.yaml:10]:
```suggestion
  InstallerSha256: 1BAC57215ADE599D1619BB1385EC74717C9261C4C00A1A24B6E2073B741F7997
  InstallerSwitches:
    Silent: /qb
    SilentWithProgress: /q
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**